netbox/extras/utils.py

```diff
@@ -67,3 +67,15 @@ def is_report(obj):
         return issubclass(obj, Report) and obj != Report
     except TypeError:
         return False
+
+
+def is_script_or_report(obj):
```

Comment:
This name is confusing, as reports will no longer exist with this change in place. IMO we should retain & expand the original `is_script()` function instead of introducing a new function.

Also, the `is_report()` function is now obsolete and can be removed.